include/clap/entry.h

```diff
@@ -29,9 +29,11 @@ extern "C" {
 // of this writing).
 //
 // Each directory should be recursively searched for files and/or bundles as appropriate in your OS
-// ending with the extension `.clap`.
+// ending with the extension `.clap`. On windows and linux, the '.clap' file is a renamed shared
+// library object, so traversing the path should stop at regular files or symlinks. On macOS
```

Comment:
Why this sentences? `so traversing the path should stop at regular files or symlinks`

Reply:
You mean that it should not follow symlinks if they are pointing to directories?

Reply:
In that case it should be "stop at regular files or symlinks to regular files ending in `.clap`".

Reply:
Stop what?

Reply:
Sorry but I've read that paragraph many times and I'm super confused by it, also I don't understand what problem it tries to solve.